cypress/support/config/settings.js

```diff
@@ -2685,6 +2827,27 @@ module.exports = () => ({
       },
       liveRadio: { environments: undefined, smoke: false },
       onDemandRadio: { environments: undefined, smoke: false },
+      onDemandTV: {
+        environments: {
+          live: {
+            paths: [
+              '/marathi/bbc_marathi_tv/tv_programmes/w13xttr2', // Brand
```

Comment:
Do we want to add a comment here saying not to enable until Marathi is up and broadcasting again?

Reply:
We _could_ also do something slightly hacky like adding this path to the errorPage404 config for marathi on test https://github.com/bbc/simorgh/blob/7986da03200151a66b98ff3e04353148f158de12/cypress/support/config/settings.js#L2801. Probably overkill, but would be an indication to us that it was now available.

Reply:
Nope, tried it and it didn't work anyway! I have put a comment stating that Marathi not currently broadcasting - do not enable.